ADALiOS/ADALiOS/ADALiOS.h

```diff
@@ -20,11 +20,11 @@
 //version in static define until we identify a better place:
 #define ADAL_VER_HIGH       2
 #define ADAL_VER_LOW        0
-#define ADAL_VER_PATCH      1
+#define ADAL_VER_PATCH      2
 
 #define STR_ADAL_VER_HIGH   "2"
 #define STR_ADAL_VER_LOW    "0"
-#define STR_ADAL_VER_PATCH  "1"
+#define STR_ADAL_VER_PATCH  "2"
```

Comment:
It's the ADAL version number. I'm trying to get in the habit of increasing it between versions.